app/assets/stylesheets/custom.scss

```diff
@@ -355,3 +355,33 @@ ul, menu, dir {
   text-align: center;
   margin-top: 120px;
 }
+
+ul.edit-listings  {
```

Comment:
Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space